Package.swift

```diff
@@ -66,6 +70,15 @@ let package = Package(
 				.plugin(name: "SwiftLint", package: "SwiftLintPlugin"),
 			]
 		),
+		.target(
+			name: "API",
+			dependencies: [
+				.AnyCodable,
```

Comment:
I think this also needs `Auth` as a dependency

Reply:
oh sure, thank you!